### ISSUE TITLE
Implement preconnect to Google fonts

### ISSUE THREAD

**Issue by leobaiano** [WordPress/twentyseventeen#105]
Add to preconnect tag to google sources. The matter was discussed in ticket [#37171](https://core.trac.wordpress.org/ticket/37171) and it is already being implemented in the themes that comes with WordPress, then I think it is important to implement this new theme as well.

The implementation should be made in the functions.php through [`wp_resource_hints();`](https://make.wordpress.org/core/2016/07/06/resource-hints-in-4-6/) function; and should only be included if the google fonts is being used, then it is interesting to use [`wp_style_is();`](https://developer.wordpress.org/reference/functions/wp_style_is/) so if a child theme developer decides to remove google preconnect the sources will not be included.

**Comment by davidakennedy** [comment]
@leobaiano Thanks for the issue and PR. Reading through the Core ticket, this seems like a smart thing to do. Let's wait on that since you're working on both. Once that patch is solid, we can make sure everything here is the same and merge.

**Comment by leobaiano** [comment]
OK, thanks very much!

Em 28/09/2016 19:52, "David A. Kennedy" notifications@github.com escreveu:

**Comment by lukecav** [comment]
Would it be worth looking at using Web Font Loader?

https://github.com/typekit/webfontloader

The issue can be if a user does have JavaScript disabled in their browser though.

**Comment by leobaiano** [comment]
@lukecav This should be dealt with in another issue as it has no direct relationship with this one, at least from what I understand. You are suggesting to use the font load to load the font via font-face when the user is with JS disabled in your browser, right? Well, that should be resolved in time to load the source.

**Comment by leobaiano** [comment]
@davidakennedy I believe the ticket [37171](https://core.trac.wordpress.org/ticket/37171) has been resolved. It has to do with the ticket [38121](https://core.trac.wordpress.org/ticket/38121) which has also been resolved.

**Comment by davidakennedy** [comment]
@leobaiano Those aren't merged yet. 😄  Is it better to wait for that?

**Comment by leobaiano** [comment]
ok 😃
